Check [WILDCARD]/testdata/test/ops_sanitizer_multiple_timeout_tests.ts
running 2 tests from [WILDCARD]/testdata/test/ops_sanitizer_multiple_timeout_tests.ts
test test 1 ... FAILED ([WILDCARD])
test test 2 ... FAILED ([WILDCARD])

failures:

test 1
AssertionError: Test case is leaking async ops.
Before:
  - dispatched: 0
  - completed: 0
After:
  - dispatched: [WILDCARD]
  - completed: [WILDCARD]
Ops:
  op_sleep:
    Before:
      - dispatched: 0
      - completed: 0
    After:
      - dispatched: [WILDCARD]
      - completed: [WILDCARD]

Make sure to await all promises returned from Deno APIs before
finishing test case.
    at [WILDCARD]

test 2
AssertionError: Test case is leaking async ops.
Before:
  - dispatched: [WILDCARD]
  - completed: [WILDCARD]
After:
  - dispatched: [WILDCARD]
  - completed: [WILDCARD]
Ops:
  op_sleep:
    Before:
      - dispatched: [WILDCARD]
      - completed: [WILDCARD]
    After:
      - dispatched: [WILDCARD]
      - completed: [WILDCARD]

Make sure to await all promises returned from Deno APIs before
finishing test case.
    at [WILDCARD]

failures:

	test 1
	test 2

test result: FAILED. 0 passed; 2 failed; 0 ignored; 0 measured; 0 filtered out ([WILDCARD])

error: Test failed
